code/game/objects/items/devices/uplinks.dm

```diff
@@ -64,6 +68,9 @@ GLOBAL_LIST_EMPTY(world_uplinks)
 			if(I.job && I.job.len)
 				if(!(I.job.Find(job)) && uplink_type != UPLINK_TYPE_ADMIN)
 					continue
+			if(I.species && I.species.len)
```

Comment:
`length()` both nullchecks and checks length. Also, we generally prefer length() over .len.
```suggestion
			if(length(I.species)
```